compiler/cpp/src/thrift/generate/t_java_generator.cc

```diff
@@ -5265,6 +5274,9 @@ THRIFT_REGISTER_GENERATOR(
     "    android_legacy:  Do not use java.io.IOException(throwable) (available for Android 2.3 and "
     "above).\n"
     "    option_type:     Wrap optional fields in an Option type.\n"
+    "    handle_runtime_exceptions:\n"
+    "                     Generated services will handle RuntimeException as "
```

Comment:
I think the following documentation gives users a little more insight into what this option does.  Is there a length limit?  I was trying to make the following short. 
```
Adjust how server handles RuntimeException (RE). Default is to close connection on RE. This enables sending TApplicationException on RE.
```